cmd/backup/archive.go

```diff
@@ -63,7 +64,11 @@ func compress(paths []string, outFilePath, subPath string, algo string) error {
 	prefix := path.Dir(outFilePath)
 	switch algo {
 	case "gz":
-		compressWriter = gzip.NewWriter(file)
+		w := pgzip.NewWriter(file)
+		if err := w.SetConcurrency(1<<16, concurrency); err != nil {
```

Comment:
If I understand https://github.com/klauspost/pgzip/blob/17e8dac29df8ce00febbd08ee5d8ee922024a003/gzip.go#L21-L25 correctly, the default block size is `1<<20`. Why did you choose a different value? I maybe missing something, but this https://github.com/offen/docker-volume-backup/issues/265#issuecomment-1703898701 also sounds as if the default should be used.

Reply:
My bad, don't know how I came up with 16 there. It's supposed to be 20. Fixed that
Also changed the compression level to 5 like mentioned in the issue #265 since Gzip default (4) seems to correspond to PGzip level 5 in terms of compression ratio. So backups don't get bigger suddenly.